homeassistant/components/stream/core.py

```diff
@@ -460,7 +461,7 @@ def stash_keyframe_packet(self, packet: Packet) -> None:
         self._packet = packet
         self._hass.loop.call_soon_threadsafe(self._event.set)
 
-    def create_codec_context(self, codec_context: CodecContext) -> None:
+    def create_codec_context(self, codec_context: VideoCodecContext) -> None:
```

Comment:
`create_codec_context` is only ever called with a `VideoCodecContext` here:
https://github.com/home-assistant/core/blob/0eea3176d6b6bf871acc7a340f748af88615637e/homeassistant/components/stream/worker.py#L536-L540